ui/src/pages/HomePage.js

```diff
@@ -54,144 +59,163 @@ export default () => {
 
   return (
     <Layout>
-      <Box bg="secondaryBackground" w="100%" pt={[4, 8]} px={[1, 24]}>
+      <Box w="100%" pt={[4, 8]} px={[1, 24]} fontFamily="Marianne" color="#1E1E1E">
         <Container maxW="xl">
-          <Breadcrumb>
+          <Box bg="#F8F8F8" color={"#D07C75"} borderLeft="4px" role="none" p={5}>
+            <Flex>
+              <Box>
+                <FontAwesomeIcon icon={faInfoCircle} color={"#D07C75"} />
+              </Box>
+              <Text color="black" fontSize="14px" px={2}>
+                Grâce à vos retours une anomalie a été détectée dans le code qui testait la présence d'un SIRET sur un
+                titre. Un correctif va être effectué très prochainement.
+                <Text fontWeight="extrabold">
```

Comment:
Un `<Text>` imbriqué dans un `<Text>` génère un `<p>` imbriqué dans un `<p>` et un paragraphe dans un paragraphe n'est pas accepté par le Strict Mode de React. Tu peux utiliser l'attribut **as** pour contourner le problème dans ce cas précis et imbriquer un span dans un paragraphe :

```suggestion
                <Text fontWeight="extrabold" as="span" >
```

Reply:
Ok merci pour l'info :)